spec/cassettes/LegalFrameworkAPI_ThresholdWaivers/_call/successful_API_call/responds_to_calling_service_with_parsed_response.yml

```diff
@@ -2,13 +2,13 @@
 http_interactions:
 - request:
     method: post
-    uri: https://legal-framework-api-staging.cloud-platform.service.justice.gov.uk/threshold_waivers
+    uri: https://ap-5044-remove-full-s8-legal-framework-uat.cloud-platform.service.justice.gov.uk/threshold_waivers
```

Comment:
will these need rerecording against staging, once this https://github.com/ministryofjustice/legal-framework-api/pull/1585

Reply:
Good shout :+1: